lib/mix/phoenix.ex

```diff
@@ -80,8 +80,8 @@ defmodule Mix.Phoenix do
       {k, {:array, _}} -> {k, []}
       {k, :belongs_to} -> {k, nil}
       {k, :integer}    -> {k, 42}
-      {k, :float}      -> {k, "120.5"}
-      {k, :decimal}    -> {k, "120.5"}
+      {k, :float}      -> {k, 120.5}
+      {k, :decimal}    -> {k, Decimal.new(43.20)}
```

Comment:
This data is meant to be sent over the form or via changesets, which can do casting. In fact, sending `Decimal.new(43.20)` may not even work. Is there a particular error you are seeing?